lib/SIL/Utils/OwnershipUtils.cpp

```diff
@@ -229,13 +229,6 @@ void BorrowingOperand::visitConsumingUsesOfBorrowIntroducingUserResults(
     // This enables one to walk the def-use chain of guaranteed phis for a
     // single guaranteed scope.
     value.visitLocalScopeEndingUses([&](Operand *valueUser) {
-      if (auto subBorrowScopeOp = BorrowingOperand::get(valueUser)) {
-        if (subBorrowScopeOp->consumesGuaranteedValues()) {
-          subBorrowScopeOp->visitUserResultConsumingUses(func);
```

Comment:
The point of this method was to be able to chain through a piece-wise guaranteed lifetime. Given that we are no longer doing that I don't think that this method makes sense as written anymore.

Reply:
I am leaving BorrowingOperand::visitConsumingUsesOfBorrowIntroducingUserResults unchanged now. It gets called transitively from BorrowedValue::areUsesWithinScope which gets used in SemanticARCOpt